store/store.go

```diff
@@ -548,7 +548,7 @@ func (s *Store) RemoveACI(key string) error {
 		dirUid := int(fi.Sys().(*syscall.Stat_t).Uid)
 
 		if uid != dirUid && uid != 0 {
-			return fmt.Errorf("permission denied, are you root or the owner of the image?")
+			return fmt.Errorf("permission denied, are you root or the owner of the image? User: uid=%d euid=%d. Image directory: uid=%d", uid, os.Geteuid(), dirUid)
```

Comment:
I'm ok landing this for failure inspection, but if reworded a bit better we could even keep it there for the future. What about the following?
```
return fmt.Errorf("permission denied on %s, directory owner id (%d) does not match current effective id (%d). Are you root or the owner of the image?", dirUid, uid, os.Geteuid())
```